xen-exporter.py

```diff
@@ -264,10 +274,13 @@ def do_GET(self):
     port = os.getenv("PORT", "9100")
     bind = os.getenv("BIND", "0.0.0.0")
 
+    if os.getenv("XEN_MODE"):
+        if os.getenv("XEN_MODE", "host") != "host" and os.getenv("XEN_MODE", "host") != "pool":
```

Comment:
Can you please update the README file to list this option and explain what it is for?

Reply:
No problem.
It's done. OK for you or you need more information ?

Reply:
Thank you for the update to the readme. With that done I think I can merge it in.